devicehive-common-auth/src/main/java/com/devicehive/resource/impl/BaseApiInfoResourceImpl.java

```diff
@@ -58,7 +58,7 @@ public BaseApiInfoResourceImpl(TimestampService timestampService) {
     }
 
     @Override
-    public Response getApiInfo(UriInfo uriInfo) {
+    public Response getApiInfo(UriInfo uriInfo, Boolean wsEnabled) {
```

Comment:
Parameter name should be `wssEnabled` or something like `secureEnabled` or `isSecure`
`wsEnabled` is confusing